workspaces/theme/plugins/theme/src/utils/createComponents.ts

```diff
@@ -157,8 +157,11 @@ export const createComponents = (themeConfig: ThemeConfig): Components => {
           },
         },
         outlined: {
+          border: `1px solid color-mix(in srgb, currentColor 50%, transparent)`,
+          borderOpacity: 0.5,
```

Comment:
This line wasn't needed, it was a testing code that I forgot to remove. Let's remove it in our next PR against theme plugin. Sorry I just found out @christoph-jerolimov